mro/helpers.py

```diff
@@ -1,4 +1,49 @@
+import json
+
+from .json_encoder import JsonEncoder
+from .mro_list import MroList
+from .mro_dict import MroDict
+
 
 def create_column_name_index_map(cursor):
     description = cursor.description
-    return {column.name: index for column, index in zip(description, range(len(description)))}
\ No newline at end of file
+    return {column.name: index for column, index in zip(description, range(len(description)))}
+
+
+def mro_objects_to_json(obj):
+    json_string = json.dumps(obj, cls=JsonEncoder)
+    return json_string
+
+
+def parse_to_mro_objects(column, instance, data):
+    # Haven't seen a way to override the top level object being an array rather than an object
+    if isinstance(data, list):
+        data = list_to_mro_list(column, instance, data)
+    elif isinstance(data, dict):
+        data = dict_to_mro_dict(column, instance, data)
+    return data
+
+
+def dict_to_mro_dict(column, instance, dict_):
+    output_dict = {}
+    for k, v in dict_.items():
+        if isinstance(v, list):
```

Comment:
As discussed, i don't think so as it would result in changing the types automatically for the user (and would result in retrieving different types vs what is originally set), this would also cause issues if the db already contains values.